package.json

```diff
@@ -17,8 +17,8 @@
     "clean": "lerna clean -y && rm -Rf node_modules",
     "check": "lerna run check --since HEAD --exclude-dependents",
     "check-ci": "lerna run check",
-    "test": "lerna run test",
-    "test-ci": "lerna run test-ci",
+    "test": "rimraf .nyc_output && lerna exec -- nyc --no-clean --cwd ../.. --reporter=none npm run test && npm run report-coverage",
```

Comment:
you may want to use some `pretest` or magic package to allow `&&` in windows

Reply:
`&&` should work just fine in npm scripts in Windows ... there are some issues with coverage generation that only seem to affect Windows, though, I'm currently looking into that